changelog.txt

```diff
@@ -3,12 +3,11 @@
 = 2.4.2 - xxx-xx-xx =
 * Fix - Action callback arguments count in ShipStation tracking integration #1841
 * Fix - Google Pay scripts loading on unrelated admin pages #1834
-* Fix - Do not ignore disabled APMs list in blocks #1865
```

Comment:
:thinking: 

This one should remain, it's a bug present in all releases with blocks.